models/article-model.js

```diff
@@ -23,4 +23,17 @@ function responseArticle(article_id) {
 })
 }
 
-module.exports = { responseAllArticles, responseArticle }
\ No newline at end of file
+function responseComments(article_id) {
+    return db.query(`SELECT comment_id, comments.votes, comments.created_at, comments.author, comments.body, comments.article_id, articles.article_id FROM articles LEFT JOIN comments ON comments.article_id = articles.article_id WHERE articles.article_id = $1 ORDER BY comments.created_at DESC`, [article_id])
+    .then(({ rows }) => {
+        if (rows.length === 0) {
+            return Promise.reject({ status:404, msg: 'article does not exist' })
+        }
+        if (rows[0].comment_id === null) {

```

Comment:
It will likely be more helpful to check the article exists first and throw an error if it doesn't in that function, then we can allow the code to continue here knowing that an empty response means no comments, not that the article already exists. It's probably fine for now, just something to think about 👍